hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyTable.tsx

```diff
@@ -100,20 +101,23 @@ const PolicyTable: FC = () => {
             <DateTimeRenderer date={DateTime.fromMillis(info.getValue() as number)} isApprox />
           </Skeleton>
         ),
-        header: t('policyTable.header.created') as string,
+        header: t('Listings.policy.header.created') as string,
       },
       {
         id: 'actions',
-        header: t('policyTable.header.actions') as string,
+        header: t('Listings.policy.header.actions') as string,
         sortingFn: undefined,
         cell: (info) => {
           return (
             <Skeleton isLoaded={!isLoading}>
-              <IconButton
-                size="sm"
-                onClick={() => navigate(`/datahub/${info.row.original.type}/id`)}
-                aria-label={t('policyTable.action.edit')}
-                icon={<FaEdit />}
+              <DataHubListAction
+                onDelete={() =>
+                  deletePolicy
+                    .mutateAsync(info.row.original.id)
+                    .then((e) => console.log('XXXXXX', e))
```

Comment:
Looks like left over.

Reply:
See https://hivemq.kanbanize.com/ctrl_board/57/cards/19819/details/

Reply:
Make sense. I will put some further details into the ticket.